src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -0,0 +1,437 @@
+| Purity                        | RECOMMENDED | [number][] | Purity of the radiolabeled compound (for example, 90).                                                                                                                                                                                                                  |
+| PurityUnit                    | RECOMMENDED | [string][] | Unit of the radiochemical purity (for example, "percent").                                                                                                           
+
+
+#### Time
+
+This section is mandatory and contains timing information about the imaging experiment. We refer to the common principles for the standards for describing dates and timestamps, including possibilities for anonymization ([LINK](https://bids-specification.readthedocs.io/en/stable/02-common-principles.html)). 
+
+| **Key name**   | **Requirement level** | **Data type**            | **Description**                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| -------------- | --------------------- | ------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| TimeZero        | REQUIRED | [string][] | Time zero to which all scan and/or blood measurements have been adjusted to, in the unit “hh:mm:ss”. This should be equal to `InjectionStart` or `ScanStart`.             |
+| ScanStart       | REQUIRED | [number][] | Time of start of scan with respect to `TimeZero` in the default unit seconds.                                                                                             |
+| InjectionStart  | REQUIRED | [number][] | Time of start of injection with respect to `TimeZero` in the default unit seconds. This corresponds to DICOM Tag (0018,1042) converted to seconds relative to `timeZero`. |
```

Comment:
```suggestion
| InjectionStart  | REQUIRED | [number][] | Time of start of injection with respect to `TimeZero` in the default unit seconds. This corresponds to DICOM Tag (0018,1042) converted to seconds relative to `TimeZero`. |
```

Modified to `TimeZero` to match the other instances

Reply:
@mnoergaard  and I agree with that change.

Reply:
@sappelhoff and @effigies can you please help here, how can we deal with these changes that are outdated?

Reply:
If it's outdated, you'll just need to apply them manually.